update-imported-docs/README-hi.md

```diff
@@ -0,0 +1,62 @@
+
+# अपडेटेड इम्पोर्टेड डॉक्स
+
+यह स्क्रिप्ट डॉक्स फ़ाइलों को अपडेट करती है जो अन्य रीपॉज़टॉरी से उत्पन्न होती हैं।
+यह अपने इनपुट के रूप में एक YAML फ़ाइल नाम को स्वीकार करता है जिसे जिसे प्रति-रेपो आधार पर अनुकूलित किया जा सकता है।।
```

Comment:
```suggestion
यह अपने इनपुट के रूप में एक YAML फ़ाइल नाम को स्वीकार करता है जिसे प्रति-रेपो आधार पर अनुकूलित किया जा सकता है।
```

Reply:
The second part of this sentence is not making any sense `जिसे जिसे प्रति-रेपो आधार पर अनुकूलित किया जा सकता है।।`.

Reply:
The English version says:- "It accepts a YAML file name as its input which can be customized on a per-repo basis"
And the second part states:- "which can be customized to per-repo basis"

Isn't that necessary? What do you think!